### PR TITLE
NOISSUE - Add README

### PR DESCRIPTION
**Pull request by dborovcanin**

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -8,3 +8,35 @@ Licensing service
...
+| MF_LICENSE_LOG_LEVEL        | License service log level | error          |
+| MF_LICENSE_DB_HOST          | DB host                   | localhost      |
+| MF_LICENSE_DB_PORT          | DB port                   | 5432           |
+| MF_LICENSE_DB_USER          | DB user                   | mainflux       |
+| MF_LICENSE_DB_PASS          | DB pass                   | mainflux       |
+| MF_LICENSE_DB               | DB name                   | license        |
+| MF_LICENSE_DB_SSL_MODE      | DB SSL mode               | disable        |
+| MF_LICENSE_DB_SSL_CERT      | DB SSL cert in PEM        |                |
+| MF_LICENSE_DB_SSL_KEY       | DB SSL key                |                |
+| MF_LICENSE_DB_SSL_ROOT_CERT | DB SSL   root cert        |                |
```
Comment (manuio) [inline README.md]: fix the double white space here `SSL   root`

**Review by manuio — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -8,3 +8,35 @@ Licensing service
...
+| MF_LICENSE_CLIENT_TLS       | Client TLS                | false          |
+| MF_LICENSE_CA_CERTS         | gRPC CA cert              |                |
+| MF_LICENSE_PORT             | service HTTP port         | 8081           |
+| MF_LICENSE_SERVER_CERT      | server TLS cert           |                |
+| MF_LICENSE_SERVER_KEY       | server TLS cert key       |                |
+| MF_JAEGER_URL               | tracing URL               |                |
+| MF_AUTH_URL                 | Auth service UR;          | localhost:8181 |
```
Comment (manuio) [inline README.md]: Fix `UR;`

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by nmarcetic — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**